game-app/game-core/src/main/java/games/strategy/triplea/util/UnitSeparator.java

```diff
@@ -127,4 +133,9 @@ private UnitSeparator() {}
     }
     return new TreeSet<>(categories.keySet());
   }
+
+  private static boolean isRelevantAirUnit(final Unit unit) {
```

Comment:
This method name is too contextual to really help. A casual reader does not know what 'relevant' is or is not. As a suggestion, `isAirWithHitPointsRemaining` would seem like a more descriptive name.

Reply:
New name will come with the next commit